ci-operator/jobs/openshift/release/openshift-release-master-periodics.yaml

```diff
@@ -19294,16 +19284,6 @@ periodics:
     job-release: "4.7"
     pj-rehearse.openshift.io/can-be-rehearsed: "true"
   name: periodic-ci-openshift-release-master-nightly-4.7-e2e-metal-assisted
-  reporter_config:
-    slack:
-      channel: '#assisted-deployment-ci'
-      job_states_to_report:
-      - failure
-      - error
-      report_template: '{{if eq .Status.State "success"}} :rainbow: Job *{{.Spec.Job}}*
-        ended with *{{.Status.State}}*. <{{.Status.URL}}|View logs> :rainbow: {{else}}
-        :volcano: Job *{{.Spec.Job}}* ended with *{{.Status.State}}*. <{{.Status.URL}}|View
-        logs> :volcano: {{end}}'
```

Comment:
Were these intentional changes?

Reply:
yes, this was an accident somehow. New force push just now to get it right and rebase on latest master